docs/tutorial/performance.md

```diff
@@ -62,14 +62,34 @@ at once, consider the [Chrome Tracing](https://www.chromium.org/developers/how-t
 Chances are that your app could be a little leaner, faster, and generally less
 resource-hungry if you attempt these steps.
 
-1. [Carelessly including modules](#1-carelessly-including-modules)
-2. [Loading and running code too soon](#2-loading-and-running-code-too-soon)
-3. [Blocking the main process](#3-blocking-the-main-process)
-4. [Blocking the renderer process](#4-blocking-the-renderer-process)
-5. [Unnecessary polyfills](#5-unnecessary-polyfills)
-6. [Unnecessary or blocking network requests](#6-unnecessary-or-blocking-network-requests)
-7. [Bundle your code](#7-bundle-your-code)
-8. [Call `Menu.setApplicationMenu(null)` when you do not need a default menu](#8-call-menusetapplicationmenunull-when-you-do-not-need-a-default-menu)
+- [Performance](#performance)
+  - [Measure, Measure, Measure](#measure-measure-measure)
+    - [Recommended Reading](#recommended-reading)
+  - [Checklist: Performance recommendations](#checklist-performance-recommendations)
+    - [1. Carelessly including modules](#1-carelessly-including-modules)
+      - [Why?](#why)
+      - [How?](#how)
+    - [2. Loading and running code too soon](#2-loading-and-running-code-too-soon)
+      - [Why?](#why-1)
+      - [How?](#how-1)
+    - [3. Blocking the main process](#3-blocking-the-main-process)
+      - [Why?](#why-2)
+      - [How?](#how-2)
+    - [4. Blocking the renderer process](#4-blocking-the-renderer-process)
+      - [Why?](#why-3)
+      - [How?](#how-3)
+    - [5. Unnecessary polyfills](#5-unnecessary-polyfills)
+      - [Why?](#why-4)
+      - [How?](#how-4)
+    - [6. Unnecessary or blocking network requests](#6-unnecessary-or-blocking-network-requests)
+      - [Why?](#why-5)
+      - [How?](#how-5)
+    - [7. Bundle your code](#7-bundle-your-code)
+      - [Why?](#why-6)
+      - [How?](#how-6)
+    - [8. Call `Menu.setApplicationMenu(null)` when you do not need a default menu](#8-call-menusetapplicationmenunull-when-you-do-not-need-a-default-menu)
+      - [Why?](#why-7)
+      - [How?](#how-7)
```

Comment:
Appreciate the fixes, but we actually don't want to include the Performance header and items like Why/How in this list as it makes it too verbose. Could you please revert this part to how it was previously?